test/internals/initialConfig.test.js

```diff
@@ -10,6 +10,8 @@ const split = require('split2')
 const deepClone = require('rfdc')({ circles: true, proto: false })
 const { deepFreezeObject } = require('../../lib/initialConfigValidation').utils
 
+process.removeAllListeners('warning')
```

Comment:
Is this gonna cause an annoying bug where other things that attach warning listeners get them removed unexpectedly by this file?

Reply:
Each test file is an independent process.

Reply:
Ah! That mitigates cross test listener fuckery, would fastify itself ever add listeners though?

Reply:
Only through the deprecation library.